UNL-CSE-AMBASSADORS/VITA
# [261] Enable Admin to Edit Volunteer Information

This PR enables an admin to edit all information of a user by hitting edit on the user management page. 

Please note that the `/management/users/` page uses an extremely old pattern of creating stuff in the PHP response and JQuery. I wanted to update this to use AngularJS and whatnot, but it would've been way too much work for the scope of this PR. I did slightly refactor the JS file though to be a little easier to work with, still not great, but it is better.

Also, I want specific comment on whether or not an admin should be allowed to edit the email of a user. Currently, users can't even edit their own emails. This specific feature request stemmed from a time when I had to change the email address of a user in the DB because an admin had accidentally typed it wrong when they added the user. However, I'm not sure enabling editing of the email without some sort of email verification (like happens during user registration) is a good idea. But I do want opinions on it. @CollinsSpencer @jjstobbe 

## Testing
1. Run the DB scripts
2. Run the `./scripts/build.sh`
3. Log in as `siteadmin@test.test`
4. Navigate to the "User Management" page
5. Hit the edit button on any user
    * A modal should pop up
        * The modal should be populated with the information of the selected user
        * Hitting cancel, the X, or outside of the modal should close the modal
6.  Change any/all of the information and hit the "Save" button
    * It should save successfully, close the modal, and the table should update
7. Change the information on another user
    * It should save successfully and not affect any other users

## Additional Testing
1. Ensure the "Add User" functionality still works properly
